outlookmsgfile.py

```diff
@@ -402,7 +408,11 @@ class UNICODE(VariableLengthValueLoader):
   @staticmethod
   def load(value, **kwargs):
     # value is a bytestring encoded in UTF-16.
-    return value.decode("utf16")
+    decoded = value.decode("utf16")
+    # do c-style strings get encoded as utf16?
+    # is there an off-by-one error in the variable length math?
+    decoded =  decoded.removesuffix('\x00')
```

Comment:
I would be curious to know more about what might be going on here.

Earlier we have a value_length field that is ignored because we use the whole embedded stream. Maybe it can't be ignored.